migration/src/jira_util.py

```diff
@@ -289,34 +289,34 @@ def repl_att(m: re.Match):
     return text
 
 
-def embed_gh_issue_link(text: str, issue_id_map: dict[str, str]) -> str:
+def embed_gh_issue_link(text: str, issue_id_map: dict[str, str], gh_number_self: int = None) -> str:
```

Comment:
I think Python itself does not enforce these type hints?

And I think I may've messed some of them up with copy/paste in past PRs.

I will try `mypy` static analysis and see what it says ;)

Reply:
The default is invalid...

Reply:
Actually, the default value is not needed. https://github.com/apache/lucene-jira-archive/commit/614a3cd62be8ecf6207e9fb682cd0b376db051fd

Reply:
@mikemccand Thanks for noticing this. I delayed checking type hits with `mypy` - will install and do static analysis.